channeld/channel_wire.csv

```diff
@@ -113,6 +116,11 @@ msgdata,channel_sending_commitsig,htlc_sigs,secp256k1_ecdsa_signature,num_htlc_s
 # Wait for reply, to make sure it's on disk before we send commit.
 msgtype,channel_sending_commitsig_reply,1120
 
+# When we reconnect and resend the commitment we also need to tell the master
+# the commitment details so it can create the penalty once we get the revoke.
+msgtype,channel_resending_commitsig,1130
+msgdata,channel_resending_commitsig,pbase,?penalty_base,
```

Comment:
No, this is in the wrong place, leading to us having to keep two values inside lightningd.

See 74c8a21a2a2d45a3ef47f58bff6c4def262495d7

Have *channeld* keep the two values, give it at the right time (when we receive the revoke), and we don't need that logic in lightningd.  We just need lightningd to save (the single, last pbase) to the db so it can hand it back to channeld when channeld restarts.